### DISCUSSION TITLE
Can't open support Ticket

### DISCUSSION THREAD

**Discussion by hsandhu01** [community/community#130062] [Other Feature Feedback, Questions, & Ideas]
When I click new ticket it just sends me back to the support page. Anyone know how to resolve this or have a similar issue?

**Comment by davevad93** [comment]
Hi @hsandhu01 ,

Try opening a ticket [here](https://support.github.com/contact/account), this link will redirect you to GitHub account support.

**Comment by hsandhu01** [comment]
@davevad93 that worked perfectly! Thank you so much. Have a great day.

**Reply by davevad93** [reply]
You're welcome @hsandhu01 , have a great day too!
